shared/config.ts

```diff
@@ -15,14 +15,16 @@ interface WalletConfig {
 class ConfigClass {
   api = {
     url: process.env.API_URL,
-    // TODO: use wallet address/signature?
-    address: process.env.API_ADDRESS,
-    signature: process.env.API_SIGNATURE,
   };
 
   signature = {
     api: process.env.API_SIGN_ADDRESS ?? '',
     transactionChecker: process.env.TRANSACTION_CHECKER_ADDRESS ?? '',
+    allowedMessages: {
+      regex: /^cfp-\d*-\d*:.*$/,
+      startsWith:
+        'By_signing_this_message,_you_confirm_to_LOCK_that_you_are_the_sole_owner_of_the_provided_Blockchain_address._Your_ID:_',
+    },
```

Comment:
Just have a list of regexs? Can use the following for the sign text:
`/By_signing_this_message,_you_confirm_to_LOCK_that_you_are_the_sole_owner_of_the_provided_Blockchain_address._Your_ID:_.*/`